metaserver/src/main/scala/scala/meta/languageserver/Formatter.scala

```diff
@@ -19,7 +19,7 @@ abstract class Formatter {
 object Formatter extends LazyLogging {
 
   /** Returns formatter that does nothing */
-  def noop: Formatter = new Formatter {
+  lazy val noop: Formatter = new Formatter {
```

Comment:
this isn't strictly related to this PR, but I think it makes sense to save some allocations